openedx/core/djangoapps/user_authn/tests/test_cookies.py

```diff
@@ -115,18 +115,15 @@ def test_set_logged_in_jwt_cookies(self):
         self._assert_recreate_jwt_from_cookies(response, can_recreate=True)
 
     @patch.dict("django.conf.settings.FEATURES", {"DISABLE_SET_JWT_COOKIES_FOR_TESTS": False})
-    def test_delete_and_are_logged_in_cookies_set(self, jwt_cookies_disabled, jwk_is_set):
-        jwt_private_signing_jwk = settings.JWT_AUTH['JWT_PRIVATE_SIGNING_JWK'] if jwk_is_set else None
-        with patch.dict("django.conf.settings.FEATURES", {"DISABLE_SET_JWT_COOKIES_FOR_TESTS": jwt_cookies_disabled}):
-            with patch.dict("django.conf.settings.JWT_AUTH", {"JWT_PRIVATE_SIGNING_JWK": jwt_private_signing_jwk}):
-                setup_login_oauth_client()
-                response = cookies_api.set_logged_in_cookies(self.request, HttpResponse(), self.user)
-                self._copy_cookies_to_request(response, self.request)
-                self.assertTrue(cookies_api.are_logged_in_cookies_set(self.request))
-
-                cookies_api.delete_logged_in_cookies(response)
-                self._copy_cookies_to_request(response, self.request)
-                self.assertFalse(cookies_api.are_logged_in_cookies_set(self.request))
+    def test_delete_and_are_logged_in_cookies_set(self):
```

Comment:
If we are disabling the `settings_override` middleware then we can avoid this change.

Reply:
how?

Reply:
Most probably this test was failing because our settings were getting override but if we are disabling that middleware then default values should apply which should pass this test.

Reply:
This is not our test. this is default `user_authn` cookies tests. This change was also missed in the conflict resolution somehow.

Reply:
also, settings middleware has nothing to do with cookies